### ISSUE TITLE
SetColumn with Tuple

### ISSUE THREAD

**Issue by wi101** [outworkers/phantom#669]
the SetColumn with Tuple parameter does not work correctly. This is an Example:
```
case class Model(id: Int, element: Set[(Int, String)])
```

```
abstract class Schema extends CassandraTable[Schema, Model] with RootConnector {

  override def tableName: String = "example"

  object id  extends IntColumn(this) with PartitionKey
  object element  extends SetColumn[(Int, String)](this)

}
```
by running CQLSH > desc example
I got this result:
```
CREATE TABLE phantom_example.example (
    id int PRIMARY KEY,
    element frozen<tuple<int, text>>
) WITH bloom_filter_fp_chance = ....
```
Instead
```
CREATE TABLE phantom_example.example (
    id int PRIMARY KEY,
    element set<frozen<tuple<int, text>>>
) WITH bloom_filter_fp_chance = ....
```

**Comment by wi101** [comment]
in com.outworkers.phantom.column.SetColumn Line 61
```
      CQLQuery(name).forcePad.append(QueryBuilder.Collections.frozen(valuePrimitive.cassandraType))
```
is missing .queryString
```  CQLQuery(name).forcePad.append(QueryBuilder.Collections.frozen(valuePrimitive.cassandraType)).queryString
```

**Comment by alexflav23** [comment]
Fixed in `2.7.4`.
